db/schema.rb

```diff
@@ -54,7 +54,7 @@
     t.index ["username", "domain"], name: "index_accounts_on_username_and_domain", unique: true, using: :btree
   end
 
-  create_table "block_mutes", force: :cascade, id: false do |t|
+  create_table "block_mutes", force: :cascade do |t|
```

Comment:
It is not good to have `id` column in `block_mutes`.
> This table will contain no data. Do not define any check constraints on this table, unless you intend them to be applied equally to all partitions. There is no point in defining any indexes or unique constraints on it, either.

[PostgreSQL: Documentation: 9.6: Partitioning](https://www.postgresql.org/docs/9.6/static/ddl-partitioning.html)